UPGRADE-3.1.md

```diff
@@ -15,6 +15,8 @@ Form
 
  * The `choices_as_values` option of the `ChoiceType` has been deprecated and
    will be removed in Symfony 4.0.
+ * Support for data objects that implements both \Traversable and \ArrayAccess in ResizeFormListener::preSubmit method has been deprecated
```

Comment:
Also, can you enclose `\Traversable`, `\ArrayAcces` and `ResizeFormListener::preSubmit` in backticks?